docs/admin-guide/install-pip.md

```diff
@@ -60,7 +58,7 @@ python3 -m venv venv
 Install Plone and a helper package, {term}`pipx`.
 
 ```shell
-venv/bin/pip install -c https://dist.plone.org/release/6.0-latest/constraints.txt Plone pipx
+venv/bin/pip install -c https://dist.plone.org/release/6.1-latest/constraints.txt Plone pipx
```

Comment:
@mauritsvanrees @davisagli which is preferred here, `6.1-latest` or `6.1.0`? I think `6.1-latest` is preferred.

Reply:
indeed 6.1-latest is good